### ISSUE TITLE
Missing keyboard navigation support when using href in DropdownMenu

### ISSUE THREAD

**Issue by baikinkirill** [gravity-ui/uikit#2091]
When specifying an href in a DropdownMenu item, keyboard support is missing. In the attached video, I am trying to click on the menu item using the Enter and Space keys.

https://github.com/user-attachments/assets/513903e3-d289-4fda-af06-44d487bdc8da

**Comment by amje** [comment]
This will be fixed in the new `Menu` component (#2106)
